### PR TITLE
iss2:: Styled components

### PR DESCRIPTION
**Pull request by krelo1320**

### CODE REVIEW

File: src/components/NameForm.tsx [hunk 1]
```diff
@@ -20,15 +58,16 @@ class NameForm extends React.Component<Props, State> {
...
-                <input
+                </Label>
+                <Input
                     className="input"
                     type="number"
                     value={this.state.value}
                     onChange={this.handleChange}/>
-                <button className="button" onClick={this.handleSubmit}>Zatwierdź</button>
+                <Button onClick={this.handleSubmit}>Zatwierdź</Button>
+                <Rotate>Hedgehog</Rotate>
```
Comment (oneacik) [inline src/components/NameForm.tsx:70]: I will for now approve that XDD

But only if it rotates symetrical.

File: src/components/NameForm.tsx [hunk 2]
```diff
@@ -1,9 +1,47 @@
 import React, {ChangeEventHandler, EventHandler, FormEventHandler, MouseEventHandler, SyntheticEvent} from "react";
 import ReactDOM from "react-dom";
+import styled, {keyframes} from "styled-components";
 
 type Props = { callback: (arg: number) => void };
 type State = { value: number };
 
+
+const Button = styled.button`
```
Comment (oneacik) [inline src/components/NameForm.tsx:9]: Those components should be in separate files for now.
please create /styled folder in components and move them there.

File: src/components/NameForm.tsx [hunk 3]
```diff
@@ -1,9 +1,47 @@
...
+  font-weight: bold;`
+
+const Label = styled.label`
+color: palevioletred;
+font-weight: bold;`
+
+const Input = styled.input`
+color: palevioletred;
+font-weight: bold;
+padding: 0.5em;
```
Comment (oneacik) [inline src/components/NameForm.tsx:25]: we will need to create constants for those values.
But it let's make it later.

File: src/components/NameForm.tsx (hunk 2 above)
Comment (oneacik) [reply src/components/NameForm.tsx:9]: reference to architecture:
https://hs3city.slack.com/archives/C018ZBP0FK5/p1604669788022000

**Comment by oneacik** [comment]
Let other see it @KKrzelowska 
Tomorrow?

File: src/components/NameForm.tsx (hunk 1 above)
Comment (krelo1320) [reply src/components/NameForm.tsx:70]: I removed it from render, but we still can use it later for some, hmm,  reasons

File: src/components/NameForm.tsx (hunk 2 above)
Comment (krelo1320) [reply src/components/NameForm.tsx:9]: Got it

File: src/components/NameForm.tsx (hunk 3 above)
Comment (krelo1320) [reply src/components/NameForm.tsx:25]: Ok

**Review by oneacik — APPROVED** [review]

**Pull request merged**